keras_nlp/models/bert/bert_masked_lm_preprocessor.py

```diff
@@ -0,0 +1,138 @@
+# Copyright 2022 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""BERT masked language model preprocessor layer."""
+
+from absl import logging
+from tensorflow import keras
+
+from keras_nlp.layers.masked_lm_mask_generator import MaskedLMMaskGenerator
+from keras_nlp.models.bert.bert_preprocessor import BertPreprocessor
+from keras_nlp.utils.keras_utils import pack_x_y_sample_weight
+
+
+@keras.utils.register_keras_serializable(package="keras_nlp")
+class BertMaskedLMPreprocessor(BertPreprocessor):
+    """BERT preprocessing for the masked language modeling task.
+    This preprocessing layer will prepare inputs for a masked language modeling
```

Comment:
Looks like the empty newlines from the version you copied from got removed. (github does this for some reason)

Can you add them back in throughout this docstring?